src/main/charts/bitbucket/values.yaml

```diff
@@ -567,6 +562,17 @@ bitbucket:
       #
       passwordSecretKey: password
 
+  shutdown:
```

Comment:
For consistency can we:

- include a general comment on what this shutdown stanza controls
- leave a carriage return after shutdown